### PR TITLE
Elrond - Rivendell Library Viewer

### PR DESCRIPTION
**Pull request by binerys**
### ELROND IS FINALLY HERE!!

To test it out, first download the following file:
[Rivendell XML](https://1drv.ms/u/s!AhR-sQyxNTqwipNi27oOhOGxOIKNiw
)

and place it in the `/database` folder. 

Also, don't forget to perform `npm install`! There are two modules that I'm using: 
* [Reactable](https://github.com/glittershark/reactable)
* [React-Spinkit](https://github.com/KyleAMathews/react-spinkit)

You should see the following at [http://localhost:3000/panel/elrond](http://localhost:3000/panel/elrond)
screen shot 2017-04-16 at 8 58 44 pm

screen shot 2017-04-16 at 9 00 14 pm

Fixes issue #76 

**Note**: I'll fix the merge conflict issues after we finalize this feature

### CODE REVIEW

**Comment by nathunsmitty** [comment]
dat boi

Nice pick for a song choice, btw :^). I'll review this soon!

**Comment by binerys** [comment]
Did some updates! 
Changed the design a bit and now it will say the last time it was updated: 
image

File: .gitignore [hunk 1]
```diff
@@ -10,3 +10,4 @@ npm-debug.log
 package.json
 public/js/react/
 data/*
+database/rivendell.xml
```
Comment (nathunsmitty) [inline .gitignore]: Newline, as per our [style guide](https://github.com/uclaradio/style-guide#file-commonalities).

File: database/rivendell.js [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+      const mtime = new Date(stats.mtime).toString();
+      
+      songList.time = mtime; 
+      callback(null, songList);
+    }); // end of parser
+  }); // end of readFile
+};
+
+
+module.exports = rivendell;
```
Comment (nathunsmitty) [inline database/rivendell.js]: Newline 🙂

File: public/panel/css/panel.css [hunk 3]
```diff
@@ -272,3 +272,56 @@ table {
   width: 100%;
 }
 
+/*** Elrond Styling ***/
```
Comment (nathunsmitty) [inline public/panel/css/panel.css]: Rather than put this in the panel CSS, would it be better to make a separate CSS file for Elrond?

**Comment by nathunsmitty** [comment]
This is so freaking cool. Amazing job, @binerys!

**Pull request merged**